tests/deployment_utils.py

```diff
@@ -220,11 +220,14 @@ def _create_temp_config(self, config: str):
             f.write(config)
         return file_path
 
-    def run(self, config: str, vcs_type: str = "none", workdir=None,
+    def run(self, config: str, vcs_type: str = "none", force_installed=False, workdir=None,
             additional_parameters="", environment=None, expected_to_fail=False):
 
-        # We cannot collect coverage from installed module
-        if utils.is_pycharm() or workdir:
+        # We can only collect coverage from local sources
+        if force_installed:
+            cmd = "python3.7 -I -m universum"
+        elif utils.is_pycharm() or workdir:
+            # Changing workdir will also lead to using installed Universum instead of local sources
```

Comment:
Changing workdir may also lead to using installed Universum instead of local sources.